HowToRPC.md

```diff
@@ -0,0 +1,165 @@
+# How to Call Google APIs: RPC Edition
+
+_For many developers, the easiest way to call a Google API is with one of our
+client libraries. But occasionally someone may prefer to make API calls
+directly - perhaps from a language or environment that we don’t support or using
+a different networking library or tool. Here we’ll show you how to do it._
+
+This page focuses on calling Google APIs directly using their underlying RPC
+interfaces. Most Google APIs are also available as REST services. For that, see
+[How to Call Google APIs, REST Edition](/HowToREST).
+
+## What you’ll need
+
+### An API definition
+
+Most Google APIs are designed as
+[RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) APIs using the
+[Protocol Buffers language](https://developers.google.com/protocol-buffers/docs/overview).
```

Comment:
I would substitute "the protocol buffers language" for just "protocol buffers".

Reply:
OK